Kentico.Kontent.Management/Models/AssetRenditions/ImageTransformation.cs

```diff
@@ -0,0 +1,39 @@
+using Kentico.Kontent.Management.Modules.ActionInvoker;
+using Newtonsoft.Json;
+
+namespace Kentico.Kontent.Management.Models.AssetRenditions
+{
+    [JsonConverter(typeof(ImageTransformationConverter))]
+    public abstract class ImageTransformation
```

Comment:
I would like to keep one file one class when it comes to models.

Reply:
Moved.